modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,93 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+$(document).ready(function() {
+
+    /* Update wai-aria on navigation */
+    $('body').on('click keydown', '.master-tabs a[role="tab"],.primary-tabs a[role="tab"],.secondary-tabs a[role="tab"],.content-tabs a[role="tab"]', function(event) {
+
+       /*
+        * Tab - Move forwards
+        * Shift and Tab - Move backwards
+        * Enter - Select that tab (PC)
+        * Space bar - Select that tab (MAC)
+        * Home - First tab
+        * End - Last tab
+        * Arrow Keys - Scroll through the tabs
+        */
+        var whitelist = [
+            'ArrowLeft',
+            'ArrowUp',
+            'ArrowDown',
+            'ArrowRight',
+            'Home',
+            'End',
+            'Tab',
+            'Shift',
+            'Enter',
+            '(Space character)',
+            'Spacebar',
+            ' '
+        ];
+        if ((event.shiftKey && event.key === 'Tab') || (event.type === 'keydown' && !whitelist.includes(event.key))) {
+            return;
+        }
+
+        var $target = $(event.currentTarget);
+        var tabName = '';
+        var tabPanel = $target.attr('data-target');
+
+        if ($target.closest('.master-tabs').length) {
+            tabName = '.master-tabs';
+        } else if ($target.closest('.primary-tabs').length) {
+            tabName = '.primary-tabs';
+        } else if ($target.closest('.secondary-tabs').length) {
+            tabName = '.secondary-tabs';
+        } else if ($target.closest('.content-tabs').length) {
+            tabName = '.content-tabs';
+        }
+
+        // Set all tabs to false
+        $(tabName + ' a').attr('aria-selected', 'false');
+        // Set all tab panels to hidden
+        $(tabName + ' div.tab-pane').attr('hidden', 'hidden');
+
+        // Add wai-aria selected on the active tab
+        $($target).attr('aria-selected', 'true');
```

Comment:
There's no need to select `$target` with `$()`, it is already a jQuery object here. Just call `$target.attr()` instead

Reply:
Fixed, I thoughtI'd I did that.